text/machine_comprehension/t5/README.md

```diff
@@ -0,0 +1,148 @@
+# T5
+
+## Use-cases
+Transformer-based language model trained on multiple tasks including summarization, sentiment-analysis, q&a, translation etc.
+The implementation in this repo is an adaptation of the [onnxt5 repo](https://github.com/abelriboulot/onnxt5) which makes the export and use of T5 with ONNX easier.
+
+## Description
+[T5](https://arxiv.org/abs/1910.10683) is a transformer model which aims to provide great flexibility and provide better semantic
+understanding through the training of multiple tasks at once.
+
+## Model
+
+ |Model        |Download  | Compressed |ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|T5-encoder       |[650.6 MB](model/t5-encoder.onnx) | [205.0 MB](model/t5-encoder.tar.gz)| 1.6 | 12 
+|T5-decoder-with-lm-head |[304.9 MB](model/t5-decoder-with-lm-head.onnx) | [304.9 MB](model/t5-decoder-with-lm-head.tar.gz)| 1.6 | 12 
```

Comment:
Can you rename your models to t5-encoder-12.onnx, t5-decoder-with-lm-head-12.onnx etc? We maintain the opsets in the file names in case we have multiple versions.